src/app/app.component.html

```diff
@@ -1 +1,7 @@
-<router-outlet></router-outlet>
\ No newline at end of file
+<router-outlet></router-outlet>
+
+<button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#exampleModal">
```

Comment:
use proper target id like 'socialShareModal' rather than exampleModalLabel, please

Reply:
updated to socialShareModalLabel